.github/workflows/pythonpackage.yml

```diff
@@ -67,6 +67,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Initial test environment setup
       run: |
+        python -m pip install -e .
```

Comment:
Why the switch to `-e`?

Reply:
Well, there wasn't an install there before, it seems it was finding pydicom just by the folder being in the current path.  I needed the install for the commands to be registered, but here was just being 'efficient' by avoiding copying all source to the site-packages location.  But it would be a better test to install separately, and avoiding the current folder, as is now the case in the release workflow.  I can update it.

Reply:
Ok, I was just curious